src/components/AssistedTagging/styles.scss

```diff
@@ -34,6 +34,20 @@ $height-assistant-options: 58px;
             flex-basis: 40%;
             margin-left: $spacing-small;
         }
+
+        .info {
+            position: absolute;
+            top: 15px;
+            right: 60px;
```

Comment:
Always declare variables for sizes and use them.

Reply:
Seconded

Reply:
Also change the right to 12px